### PR TITLE
Onboardingreviewchanges

### PR DESCRIPTION
**Pull request by AnjaliBhamaSasikumar**
Created Constructors, Hooks and POM in Onboarding task

### CODE REVIEW

File: MarsTest/NotForTesting/AllTestCase.feature [hunk 1]
```diff
@@ -1,174 +1,170 @@
...
-#	Then The record '<Language>' and '<Level>' should add to the profile page
-#	
-#	Examples: 
-#	| Language  | Level            |
-#	| Malayalam | Native/Bilingual |
-#	| English   | Fluent           |
-#	| Tamil     | Conversational   |
-#	| Hindi     | Basic            |
-
+#Scenario Outline: Testcase_2 Add langauge details
```
Comment (amanbansalnz) [inline MarsTest/NotForTesting/AllTestCase.feature:25]: why are these commented out?

File: MarsTest/NotForTesting/AllTestCase.feature [hunk 2]
```diff
@@ -306,26 +302,26 @@ Scenario: Testcase_7 Delete existing skills
 #   | Amrita School of Engineering | Denmark    | M.Tech    | Power Elecctronics | 2012 |
 #   | LBSITW                       | Bangladesh | B.Tech    | Instrumentation    | 2006 |
 #   | NSS College                  | India      | PHD       | Electronics        | 2020 |
-
+#
 #Scenario: Testcase_8.4 Cancel adding education details
 #  Given I successfully SignIn to the Website
```
Comment (amanbansalnz) [inline MarsTest/NotForTesting/AllTestCase.feature:307]: why are these commented out?

File: MarsTest/NotForTesting/SampleStepDefinitions.cs [hunk 3]
```diff
@@ -0,0 +1,155 @@
+//using MarsTest.Pages;
+//using MarsTest.Utilities;
+//using NUnit.Framework;
+//using System;
+//using System.Linq;
+//using TechTalk.SpecFlow;
+
+//namespace MarsTest.NotForTesting
+//{
+//    [Binding]
```
Comment (amanbansalnz) [inline MarsTest/NotForTesting/SampleStepDefinitions.cs:10]: why is this commented out?

File: MarsTest/POM/ShareSkillPage.cs [hunk 4]
```diff
@@ -1,59 +1,62 @@
...
-            SubCategory.SelectByValue("QA");
-            TagsText.Click();
-            TagsText.SendKeys("#TestAnalyst");
-            TagsText.SendKeys(Keys.Enter);
-            TagsText.SendKeys("#Automation");
-            ServiceType.Click();
+﻿//using OpenQA.Selenium.Support.UI;
+//using SeleniumExtras.PageObjects;
+//using System;
+//using System.Collections.Generic;
```
Comment (amanbansalnz) [inline MarsTest/POM/ShareSkillPage.cs:4]: why is this commented out?

File: MarsTest/POM/SignInPage.cs [hunk 5]
```diff
@@ -1,35 +1,35 @@
...
+﻿//using SeleniumExtras.PageObjects;
 
-namespace MarsTest.POM
-{
-    public class SignInPage 
-    {
-        IWebDriver driver;
+//namespace MarsTest.POM
+//{
+//    public class SignInPage
```
Comment (amanbansalnz) [inline MarsTest/POM/SignInPage.cs:5]: why is this commented out?

File: MarsTest/Pages/SignIn.cs [hunk 6]
```diff
@@ -10,32 +10,38 @@
...
+        IWebElement PasswordTextBox => driver.FindElement(By.XPath("/html/body/div[2]/div/div/div[1]/div/div[2]/input"));
+
+        public void SignInActions()
         {
             // open chrome browser
-            driver.Manage().Window.Maximize();
-            driver.Navigate().GoToUrl("http://localhost:5000/Home");
+            
+
+            //driver.Manage().Window.Maximize();
```
Comment (amanbansalnz) [inline MarsTest/Pages/SignIn.cs:26]: why is this commented out?

File: MarsTest/Pages/SignIn.cs [hunk 7]
```diff
@@ -10,32 +10,38 @@
...
+            //driver.Navigate().GoToUrl("http://localhost:5000/Home");
 
-            //Identify SignIn button and open
-            IWebElement SignInButton = driver.FindElement(By.XPath("//*[@id='home']/div/div/div[1]/div/a"));
-            SignInButton.Click();
             try
             {
+                //Identify SignIn button and open
```
Comment (amanbansalnz) [inline MarsTest/Pages/SignIn.cs:31]: why is this commented out?

File: MarsTest/Pages/SkillPage.cs [hunk 8]
```diff
@@ -8,77 +8,84 @@
...
-            IWebElement AddSkill = driver.FindElement(By.XPath("//body[1]/div[1]/div[1]/section[2]/div[1]/div[1]/div[1]/div[3]/form[1]/div[3]/div[1]/div[2]/div[1]/div[1]/span[1]/input[1]"));
+            //Identify and click add skill button
             AddSkill.Click();
         }
-        public string createskills(IWebDriver driver)
+
+        public string createskills()
         {
             Thread.Sleep(3000);
             //WaitHelpers.WaitToExist(driver, 3, "XPath", "//body/div[@id='account-profile-section']/div[1]/section[2]/div[1]/div[1]/div[1]/div[3]/form[1]/div[3]/div[1]/div[2]/div[1]/table[1]");
```
Comment (amanbansalnz) [inline MarsTest/Pages/SkillPage.cs:43]: why is this commented out?

File: MarsTest/StepDefinition/ProjectStepDefinitions.cs [hunk 9]
```diff
@@ -1,155 +1,161 @@
...
+
+        public ProjectStepDefinitions(Hooks Hooks)
+        {
+            this._Hooks = Hooks;
+            SignInObj = new SignIn();
+            HomePageObj = new HomePage();
+            DescriptionPageObj = new DescriptionPage();
+            LanguagePageObj = new LanguagePage();
+            SkillPageObj = new SkillPage();
+            //driver = new ChromeDriver();
```
Comment (amanbansalnz) [inline MarsTest/StepDefinition/ProjectStepDefinitions.cs:28]: why is this commented out?

File: MarsTest/Test/ShareSkill.cs [hunk 10]
```diff
@@ -1,23 +1,23 @@
...
-    {
-        [Test]
-        public void TestShareSkill()
-        {
-            var SignInPage = new SignInPage(driver);
-            SignInPage.NavigateToShareSkillHomePage();
-        }
-    }
-}
+//namespace MarsTest.Test
```
Comment (amanbansalnz) [inline MarsTest/Test/ShareSkill.cs:11]: why is this commented out?

File: MarsTest/Utilities/CommonDriver.cs [hunk 11]
```diff
@@ -6,20 +6,19 @@ namespace MarsTest.Utilities
 {
     public class CommonDriver
     {
-        public IWebDriver driver;
+        public static IWebDriver driver;
+        
+        //public void Open()
+        //{
+        //    driver = new ChromeDriver();
+        //    driver.Manage().Window.Maximize();
```
Comment (amanbansalnz) [inline MarsTest/Utilities/CommonDriver.cs:14]: why is this commented out?

Comment (amanbansalnz) [reply MarsTest/Utilities/CommonDriver.cs:14]: so can you please removed commented code out as you should never check commented code as this is seen as bad practice. Over all your other changes look fine, once you remove the commented out code will approve

Comment (AnjaliBhamaSasikumar) [reply MarsTest/Utilities/CommonDriver.cs:14]: I was doing trial and error method, will delete all uncommented codes and folders not in use

**Pull request merged**